docs/style-guide-for-curriculum-challenges.md

```diff
@@ -123,8 +123,19 @@ Here are specific formatting guidelines for challenge text and examples:
 
 - Language keywords go in `<code>` tags. For example, HTML tag names or CSS property names
 - The first instance of a keyword when it's being defined, or general keywords (i.e. "object" or "immutable") go in `<dfn>` tags
-- Single line code examples go in `<code>` tags
-- Multi-line code examples go in `<blockquote>` tags, and use the `<br>` tag to separate lines. For HTML examples, remember to use escape characters to represent the angle brackets
+- Small pieces of code examples go in `<code>` tags. See example below:
+````
+<code>thisIsAnExampleFunction()</code>
+````
+- Multi-line code examples go between two sequences of triple backticks. See example below:
```

Comment:
```suggestion
- Multi-line code blocks **must be preceded by an empty line**.  The next line must start with three backticks followed immediately by one of the [ supported languages](https://prismjs.com/#supported-languages).  To complete the code block, you must start a newline only has three backticks.

**Note:** If you are going to use an example code in YAML, use `yml` instead of `yaml`.

See example below:
```

Reply:
The tests are parsed by looking for `yml` blocks and incorporating them.  This means it should be the other way around. 'yaml' is fine, 'yml' is not.

Reply:
@ojeytonwilliams Oops!  I have updated the suggestion above.